README.md

```diff
@@ -68,7 +68,13 @@ And can be installed by using the 'install' target:
 
 ` cmake --build . --config Release --target install ` **OR** ` make install `
 
-This will install the library to the `CMAKE_INSTALL_PREFIX` path that was set. 
+This will install the library to the `CMAKE_INSTALL_PREFIX` path that was set.
+
+Alternatively, to build the library using the 'Debug' configuration:
+
+` cmake --build . --config Debug `
```

Comment:
That will only build debug for Visual Studio projects. `CMAKE_BUILD_TYPE` needs to be set to `Debug`.

Reply:
I removed that line and referred to the `CMAKE_BUILD_TYPE` variable instead. If there's a better way of explaining it, feel free to add a suggestion.